Makefile

```diff
@@ -25,6 +25,7 @@ TELEPORT_DEBUG ?= no
 GITTAG=v$(VERSION)
 BUILDFLAGS ?= $(ADDFLAGS) -ldflags '-w -s'
 CGOFLAG ?= CGO_ENABLED=1
+GO_LINTERS := "unused,govet,typecheck,deadcode,goimports"
```

Comment:
I think that using `:=` rather than `?=` here means that you can't override `GO_LINTERS` via environment variable? Not sure if that's deliberate/desired behaviour.

Reply:
Ahh, so that's what `?=` does. Makes sense, changed.